sergmiller/src/main/java/ru/fizteh/fivt/students/sergmiller/twitterStream/TwitterStream.java

```diff
@@ -0,0 +1,274 @@
+     */
+    public static void printTwitterStream(
+            final JCommanderParser jCommanderParsed) {
+        twitter4j.TwitterStream twStream = twitter4j
+                .TwitterStreamFactory.getSingleton();
+
+
+        StatusListener listener = new StatusAdapter() {
+            @Override
+            public void onStatus(final Status status) {
+                if (!jCommanderParsed.isHideRetweets() || !status.isRetweet()) {
+                    printTweet(status, jCommanderParsed);
+                }
+            }
+        };
+
+     //   FilterQuery query = new FilterQuery(jCommanderParsed.getQuery());
+        String[] queries = jCommanderParsed
+                .getQuery().toArray(
+                        new String[jCommanderParsed.getQuery().size()]);
+        twStream.addListener(listener);
+        twStream.filter(new FilterQuery().track(queries));
+      //  twStream.filter(new Filter.
+    }
+
+    /**
+     * Mod with print limited quantity of text.
+     * @param jCommanderParsed is class with query's info
+     * @throws TwitterException is kind of exception
+     */
+    public static void printTwitterLimited(
+            final JCommanderParser jCommanderParsed) throws TwitterException {
+        Twitter twitter = TwitterFactory.getSingleton();
+
+        Query query = new Query(String.join(" ", jCommanderParsed.getQuery()));
+
+        query.setCount(jCommanderParsed.getLimit());
+
+        QueryResult request = twitter.search(query);
+
+        List<Status> tweets = request.getTweets();
+
+        Boolean existTweets = false;
+
+        for (Status status: tweets) {
+            if (!jCommanderParsed.isHideRetweets() || !status.isRetweet()) {
+                printTweet(status, jCommanderParsed);
+                existTweets = true;
+            }
+        }
+
+        if (!existTweets) {
+            System.out.println("\nПо запросу "
+                    + String.join(" ", jCommanderParsed.getQuery())
+                    + " для "
+                    + jCommanderParsed.getLocation()
+                    + " ничего не найдено=(\n\n"
+                    + "Рекомендации:\n\n"
+                    + "Убедитесь, что все слова написаны без ошибок.\n"
+                    + "Попробуйте использовать "
+                    + "другие ключевые слова.\n"
+                    + "Попробуйте использовать "
+                    + "более популярные ключевые слова."
+            );
+        }
+    }
+
+    /**
+     * Getting time info.
+     * @param date is current time
+     * @return new string with special format
+     */
+    public static String getTime(final Date date) {
+        return date.toString();
+    }
+
+    /**
+     * mods and ranges for getting correct declension form.
+     */
+    public static final int MOD100 = 100;
+    public static final int MOD10 = 10;
+    public static final int LEFTBOUNDMOD100IS10 = 10;
+    public static final int RIGHTBOUNDMOD100IS20 = 20;
```

Comment:
Use ALL_UPPER_CASE for your named constants, separating words with the underscore character. For example, use TAX_RATE rather than taxRate or TAXRATE .

https://www.cwu.edu/~gellenbe/javastyle/constant.html